components/ide/jetbrains/backend-plugin/build.sh

```diff
@@ -12,3 +12,5 @@ else
 fi
 ./gradlew -PsupervisorApiProjectPath=components-supervisor-api-java--lib/ -PgitpodProtocolProjectPath=components-gitpod-protocol-java--lib/ -PenvironmentName="$JB_QUALIFIER" buildPlugin
 unzip ./build/distributions/gitpod-remote.zip -d ./build
+
+# force rebuild
```

Comment:
Reminder: Line added just to force the rebuild of backend-plugin image. We'll remove it before merging this PR into main branch.